_includes/cohort-loop.html

```diff
@@ -0,0 +1,41 @@
+{% assign all_participants = '' %}
+{%- assign all_mentors = '' -%}
+{%- assign all_experts = '' -%}
+{%- assign all_facilitators = '' -%}
+{%- assign all_projects = '' -%}
+{%- assign cohorts = site.data.cohorts | sort -%}
+
+{%- for cohort in cohorts -%}
+  {%- assign cohort_name = cohort[0] -%}
+  {%- assign experts = site.data.cohorts[cohort_name].metadata.experts | uniq | size -%}
```

Comment:
```suggestion
  {%- assign experts = cohort[1].metadata.experts | uniq | size -%}
```

Reply:
Would that work? If so, it is cleaner and please apply everywhere below. Thanks

Reply:
It does work...and you're right. This is much cleaner. Thanks. 👍